tests/common/plugins/conditional_mark/tests_mark_conditions_platform_tests.yaml

```diff
@@ -71,6 +71,30 @@ platform_tests/api/test_chassis.py::TestChassisApi::test_get_watchdog:
     conditions:
       - "'sw_to3200k' in hwsku"
 
+platform_tests/api/test_chassis.py::TestChassisApi::test_get_model:
+  skip:
+    reason: "Testcase consistently fails, raised issue to track"
+    conditions:
+      - "https://github.com/sonic-net/sonic-mgmt/issues/6512 and 'Celestica-DX010-C32' in hwsku"
```

Comment:
@assrinivasan This command may not work. It's better to use like this, then script is able to decode https from the conditions:
    conditions:
      - "hwsku in ['Celestica-DX010-C32']"
      - https://github.com/sonic-net/sonic-mgmt/issues/6512

Reply:
Modified per your review comments in latest commit, thanks!